hardware/raspberrypi/bootmodes/nvme.md

```diff
@@ -54,14 +54,13 @@ File read: pieeprom.bin
 Second stage boot server done
 ```
 
-### Firmware and Kernel
-
-At the time of writing you need to use pre-release versions of the VideoCore firmware and kernel and these need to be on the NVMe disk. When this software is officially released you will just be able to write the software to the NVMe disk directly with the rpi-imager app. Until then the easiest way to do this is to...
+### Firmware and kernel
 
-* Boot the board with a blank SSD plugged into the PCIe slot
-* Use [piclone](https://github.com/raspberrypi-ui/piclone) to copy the image to the NVMe disk. When using piclone choose the option to use "New Partition ids"
-* Update the software on the NVMe disk with the latest pre-release software using [rpi-update](../../../raspbian/applications/rpi-update.md).
+You must load pre-release versions of the VideoCore firmware and kernel to the NVMe disk. When this software is officially released you can simply write the software to the NVMe disk directly with the rpi-imager app: until then the easiest way to perform the update is:
 
+1. Boot the CM4 with a blank SSD connected to the PCIe slot
+1. Use the `SD Card Copier` application on the desktop to copy the running OS image to the NVMe disk: be sure to enable the "new partition ids" option.
+1. Update the firmware and kernel on the NVMe disk with the latest pre-release versions using [rpi-update](../../../raspbian/applications/rpi-update.md):
```

Comment:
Perhaps "with the latest" would sound better as "to the latest"?

Reply:
Yes - your wording is more gramatically correct. Fixed.